wgpu-core/src/device/life.rs

```diff
@@ -418,20 +418,45 @@ impl<B: GfxBackend> LifetimeTracker<B> {
         let remove_list = framebuffers
             .keys()
             .filter_map(|key| {
-                let mut last_submit: SubmissionIndex = 0;
+                let mut last_submit = None;
+                let mut needs_cleanup = false;
```

Comment:
these two flags are redundant, we could treat `last_submit.is_some()` as `needs_cleanup`

Reply:
They are not redundant, the framebuffer can need cleanup without the views being in any submission, in that case the submission index will be considered "0" as previously, but this cannot be the default value because the loop tries to find the lowest one.